promise/Promise.ts

```diff
@@ -92,6 +92,11 @@ declare global {
          * @param options - TrackOptions, or simply a message string.
          */
         track(options: TrackOptions | string): Promise<T>;
+
+        /**
+         * @internal
```

Comment:
`@internal - set by FetchService to relay correlation IDs on responses to downstream error handling and tracking`